scripts/create-common-bundle/index.js

```diff
@@ -20,9 +20,8 @@ const ignored_extensions = ['.d.ts', '.map'];
 const ignored_directories = ['.svelte-kit', 'node_modules/.bin', 'node_modules/rollup/dist/shared'];
 
 const ignored_files = new Set([
-	'node_modules/rollup/dist/es/rollup.browser.js',
-	'node_modules/rollup/dist/rollup.browser.js',
-	'node_modules/svelte/compiler.js'
+	'node_modules/svelte/compiler.js',
+	'node_modules/@esbuild/darwin-arm64/bin/esbuild'
```

Comment:
why did you add this? It's failing the build.

Reply:
huh, I didn't think it was necessary, I thought we only needed the wasm binary. worked locally. the bundle is kinda chunky, was trying to slim it down

Reply:
My guess is that this is what esbuild chooses for you locally to include but on Vercel it chooses something else. Maybe we need some kind of wildcard there.

Reply:
ah okay, it installs a different binary on vercel (obviously, duh) — will need to change this to use a regex instead